wordpress-mobile/WordPress-iOS
# Issue/8914 async posting continue editing

Fixes #8914 

This PR allows the Editor interface to restore the attachment upload interface when going back to post that are marked for async upload.

For non async posting it also improves the code for connecting back attachment when switching between HTML mode and Visual while uploads are ongoing.

To test:
 - Start a new post
 - Add new media to it
 - While the media is uploading tap to async upload
 - While the upload of media are still going, open back the editor and see if they are being processed correctly
 - Try the same by just switching between HTML and Visual mode while uploads are ongoing.
